src/transformers/models/egt/configuration_egt.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+EGT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    # pcqm4mv1 now deprecated
+    "graphormer-base": "https://huggingface.co/clefourrier/graphormer-base-pcqm4mv2/resolve/main/config.json",
+    # See all Graphormer models at https://huggingface.co/models?filter=graphormer
+}
+
+
+class EGTConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`~EGTModel`]. It is used to instantiate an EGT
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the EGT
+    [graphormer-base-pcqm4mv1](https://huggingface.co/graphormer-base-pcqm4mv1) architecture.
```

Comment:
Why is a Graphormer your default pretrained model? I'm quite sure the architectures are not even compatible